### DISCUSSION TITLE
Issue with FSR and the game

### DISCUSSION THREAD

**Discussion by Willy386** [Blinue/Magpie#38] [Q&A]
Hello. It is a very interesting tool. It is good to be able to use FSR technology in any game. However, im having an issue with the application. I get an error message every time i click on the button to apply Alt+F11 after 5 seconds. Message says something in Chinese related to WinRT. If i try to press Alt+F11 manually, the window of the game will not escale. 

However, if i click in the option GDI, i can expand the window, but it is not working with some applications and i get black screen. Is it normal? Am i missing something?

Thank you!

**Comment by Blinue** [comment]
Can you provide more information, such as error message and your OS version

**Reply by Willy386** [reply]
i get this error: 创建全屏窗口出错:WinRT 铀时惊锋潜枢.  i am using w10 enterprise ltsc, 1809. 64 bit. im using cpu intel i7 6700k and gpu asus dual gtx 1070. direct x 12. my screen is 4k. Thank you.

**Reply by Blinue** [reply]
#37 will help you.

**Reply by Willy386** [reply]
thank you. i have 1809 version
